setup.cfg

```diff
@@ -1,10 +1,66 @@
 [metadata]
-package_name = jwst
-description = JWST Pipeline
-author = STScI
-license = BSD
-edit_on_github = False
-github_project = spacetelescope/jwst
+name = jwst
+description = Python library for science observations from the James Webb Space Telescope
```

Comment:
I've never really liked this description. It's far too vague, in my opinion (but then so is the package name "jwst"). We're definitely not the only package out there that contains software for science observations from JWST. Our nextdoor neighbors in DATB have pkgs that contain just as much code for JWST science observations from JWST. What's distinctive about ours is the fact that it's a package of software for *calibrating* science observations from JWST.

Reply:
Ok, so something like `Library for calibration of science observations from the James Webb Space Telescope`?